### ISSUE TITLE
configurator produces bad bin file for k-type

### ISSUE THREAD

**Issue by alexeiz** [kiibohd/configurator#82]
I uploaded my json config to the web configurator and dowloaded the "latest" firmware.  Then flashed the bin file using the Windows desktop configurator.  Flashing succeeded, but the keyboard would not respond after that.  Although, it's still possible to put the keyboard into the flashing mode and flash the previous version of kiibohd.dfu.bin (which I luckily saved).  Same thing happens if I use the desktop configurator to obtain the bin file.  My previous version of the bin file which still works was created on Jan 23 2018.

**Comment by alexeiz** [comment]
here's my layout json config file: https://pastebin.com/NG4eWsLR

**Comment by InfinityTotality** [comment]
Importing that json into the desktop configurator actually works on my k-type. I have noticed that there seems to be a limit on how big a config can be though. If you delete one of those animations, it may work for you.

**Comment by alexeiz** [comment]
I can import json without a problem.  The problem is with the bin file  generated for that configuration.  When I flash that bin file to the keyboard, the keyboard doesn't work: no keys register.

**Comment by InfinityTotality** [comment]
Yes, I mean that I am able to compile a bin file, flash it to my k-type, and have it boot and function using your configuration.

**Comment by james-cosgrove** [comment]
I'm also having the same issue on the desktop configurator for my Infinity 60%. The .bin file I currently have that was made using the  old web configurator still works, but if I create a new layout and flash it, none of the keys work.

**Comment by haata** [comment]
@james-cosgrove which Infinity 60% do you have? Did it have optional LEDs?
